test/test_autograd.py

```diff
@@ -3197,14 +3197,19 @@ def fn(*inputs):
                     f_args_variable = (self_variable,) + args_variable
                     f_args_tensor = (self_tensor,) + args_tensor
                     # could run the gradchecks again, but skip since we did it for the methods above.
+                    run_gradcheck = exclude_tensor_method(name, test_name) and not is_inplace and name not in EXCLUDE_GRADCHECK
```

Comment:
the comment above is out of date now.  Also, why do you need `not is_inplace`?

Reply:
I can't run gradcheck for in_place functions here, because I will get "a leaf Variable that requires grad has been used in an in-place operation"